index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/index.css">
+    <script src="js/index.js" type="module" async></script>
+    <title>Creative Commons Global Summit 2015</title>
+  </head>
+  <body>
+    <main id="main" class="container-fullwidth">
+      <header>
+        <nav id="nav" class="container-fullwidth navbar">
```

Comment:
- [ ] Please put the `<header>` out of the `<main>.` It's not a good practice to place `<header>` inside the `<main>` because they are intended to be at the same level of hierarchy. The main tag should only be responsible for the main /hero section of the webpage.